test/integration/targets/filter_urls/jinja26.yml

```diff
@@ -0,0 +1,25 @@
+- hosts: testhost
+  gather_facts: True
```

Comment:
Disable fact gathering unless it's needed for the test. If it is needed, add a comment explaining why.